### DISCUSSION TITLE
From DateTime to DateTimeImmutable ?

### DISCUSSION THREAD

**Discussion by vinceAmstoutz** [rectorphp/rector#8498] [Q&A] [answered]
Any `rector` rule to convert dates from `DateTime` to `DateTimeImmutable` ?

**Comment by samsonasik** [comment, marked as answer]
You should can use `RenameClassRector` for that, see https://getrector.com/demo/0b284aa5-e1c5-43d1-a076-3d16f925aa2d

**Reply by vinceAmstoutz** [reply]
Tanks!
